usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
+):
+    """
+    Loading elevation raster data from GeoTIFF file.
```

Comment:
Per
https://numpydoc.readthedocs.io/en/latest/format.html#short-summary

this should be
```
"""Load elevation raster data from GeoTIFF file.

...
```

Reply:
Done